[{"count": 0, "tags": [], "bug_id": 29913, "attachment_id": null, "id": 60136, "time": "2004-07-05T05:07:56Z", "creator": "takeichim@nttdata.co.jp", "creation_time": "2004-07-05T05:07:56Z", "is_private": false, "text": "In Distributed Testing,\nI used one JMeter controller and some JMeter servers.\nIt worked well.\n\nBut after JMeter servers were restarted,\nthey became to generate no load.\nAnd JMeter controller threw java.rmi.ConnectException.\n\n\nAs far as I can see,\nJMeter controller caches remote engine stubs\ndownloaded from JMeter servers.\nAnd the stubs are never updated.\n [org.apache.jmeter.gui.action.RemoteAction]\n\nSo after a JMeter server is restarted,\nan old cached stub cannot connect to it."}, {"count": 1, "tags": [], "bug_id": 29913, "attachment_id": null, "text": "I made a patch for the above problem.\nIf this patch is applied, \na JMeter server can generate loads even after it is restarted.\n\n[change]\n ClientJMeterEngine try to download a new stub,\n when it catches ConnectException \n by calling reset() method of a remote stub.\n\n\nCould you use this patch?\nThank you.\n\n", "id": 60326, "time": "2004-07-09T03:28:48Z", "creator": "takeichim@nttdata.co.jp", "creation_time": "2004-07-09T03:28:48Z", "is_private": false}, {"count": 2, "tags": [], "bug_id": 29913, "attachment_id": 12064, "id": 60327, "time": "2004-07-09T03:35:24Z", "creator": "takeichim@nttdata.co.jp", "creation_time": "2004-07-09T03:35:24Z", "is_private": false, "text": "Created attachment 12064\npatch file for ClientJMeterEngine (nightly build)"}, {"count": 3, "tags": [], "bug_id": 29913, "attachment_id": null, "id": 60352, "time": "2004-07-09T11:23:12Z", "creator": "sebb@apache.org", "creation_time": "2004-07-09T11:23:12Z", "is_private": false, "text": "Change committed to HEAD.\n\nPlease re-open call if there are further problems."}]